config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml

```diff
@@ -18,7 +18,7 @@ postsubmits:
         - bash
         args:
         - -c
-        - apt update && apt install -y jq && scripts/test-infra/test-e2e.sh
+        - apt update && apt install -y jq && scripts/test-infra/test-e2e-postsubmit.sh
```

Comment:
I don't think we have renamed this script

Reply:
Yeah, but I was planning to do it on NFD sight. But it doesn't make sense now anyways since we are moving away from presubmit to periodic in https://github.com/kubernetes/test-infra/pull/27877

Reply:
let's change this to reflect the file being added on the PR 
`scripts/test-infra/test-e2e-presubmit.sh`